mmv1/templates/terraform/examples/backup_dr_backup_plan.tf.tmpl

```diff
@@ -0,0 +1,163 @@
+    time_zone             = string
+    start_hour_of_day     = number
+    end_hour_of_day       = number
+    days_of_week          = list(string)
+    days_of_month         = list(number)
+    week_day_of_month     = object({
+      week_of_month = string
+      day_of_week   = string
+    })
+    months = list(string)
+  }))
+  
+  # Default backup rules
+  default = [
+    {
+      rule_id               = "rule-1"
+      backup_retention_days = 5
+      recurrence_type       = "HOURLY"
+      frequency             = 6
+      time_zone             = "UTC"
+      start_hour_of_day     = 0
+      end_hour_of_day       = 24
+      days_of_week          = []
+      days_of_month         = []
+      week_day_of_month = {
+        week_of_month = ""
+        day_of_week   = ""
+      }
+      months = []
+    },
+    {
+      rule_id               = "rule-2"
+      backup_retention_days = 10
+      recurrence_type       = "DAILY"
+      frequency             = 1
+      time_zone             = "UTC"
+      start_hour_of_day     = 0
+      end_hour_of_day       = 24
+      days_of_week          = []
+      days_of_month         = []
+      week_day_of_month = {
+        week_of_month = ""
+        day_of_week   = ""
+      }
+      months = []
+    },
+    {
+      rule_id               = "rule-3"
+      backup_retention_days = 15
+      recurrence_type       = "WEEKLY"
+      frequency             = 1
+      time_zone             = "UTC"
+      start_hour_of_day     = 0
+      end_hour_of_day       = 24
+      days_of_week          = ["MONDAY", "WEDNESDAY", "FRIDAY"]
+      days_of_month         = []
+      week_day_of_month = {
+        week_of_month = ""
+        day_of_week   = ""
+      }
+      months = []
+    },
+    {
+      rule_id               = "rule-4"
+      backup_retention_days = 20
+      recurrence_type       = "MONTHLY"
+      frequency             = 1
+      time_zone             = "UTC"
+      start_hour_of_day     = 0
+      end_hour_of_day       = 24
+      days_of_week          = []
+      days_of_month         = [1, 15, 30]
+      week_day_of_month = {
+        week_of_month = "FIRST"
+        day_of_week   = "SUNDAY"
+      }
+      months = []
+    },
+    {
+      rule_id               = "rule-5"
+      backup_retention_days = 30
+      recurrence_type       = "YEARLY"
+      frequency             = 1
+      time_zone             = "UTC"
+      start_hour_of_day     = 0
+      end_hour_of_day       = 24
+      days_of_week          = []
+      days_of_month         = []
+      week_day_of_month = {
+        week_of_month = "FIRST"
+        day_of_week   = "SUNDAY"
+      }
+      months = ["JANUARY", "DECEMBER"]
+    }
+  ]
+}
+
+resource "google_backup_dr_backup_vault" "my_backup_vault" {
+  provider = google-beta
+  location = "us-central1"
+  backup_vault_id = "bv-test1"
+  description     = "This is a backup vault built by Terraform."
+  backup_minimum_enforced_retention_duration = "100000s"
```

Comment:
For this resource, it looks like only `location`, `backup_vault_id`, and `backup_minimum_enforced_retention_duration` are required. Could we removed the other fields (unless they are necessary for the test? Same for you other example.